scripts/local-training-e2e.sh

```
@@ -0,0 +1,1133 @@
+    log_step "Running training steps via port-forward..."
+
+    export KUBECONFIG="$KUBECONFIG_PATH"
+
+    # Start port-forward in background
+    log_info "Starting port-forward to training pod..."
+    kubectl port-forward -n "$NAMESPACE" "pod/$POD_NAME" 8001:8000 &
+    PF_PID=$!
+    sleep 3
+
+    # Cleanup on exit
+    trap "kill $PF_PID 2>/dev/null" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)